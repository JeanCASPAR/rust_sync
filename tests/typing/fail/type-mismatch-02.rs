use rust_sync::sync;

sync! {
    #![pass(1)]
    
    node oui(hello : int, bye : bool) = (oh)
    where
        oh : int = if bye { hello } else { 3.0 };
}

fn main() {}
