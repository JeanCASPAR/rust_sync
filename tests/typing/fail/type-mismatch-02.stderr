error: type mismatch: left has type `int`, right has type `float`
 --> tests/typing/fail/type-mismatch-02.rs:8:20
  |
8 |         oh : int = if bye { hello } else { 3.0 };
  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
